34-chinese-pangolin/technical_specification.md

```diff
@@ -0,0 +1,83 @@
+# File Deletion and Outbox Cache Strategy (Chinese Pangolin)
+**Epic Type:** Implementation Epic
+
+## Scope
+### Outline:
+
+This Epic adds a new service, which provides a deletion endpoint for files.
+Calling this enpoint internally publishes an event.
+All services subscribing to this event will then delete this file from their databases and from S3 buckets they have write access to.
+
+### Included/Required:
+
+This Epic includes:
+
+- The creation of a new service, the **Purge Controller Service (PCS)**, which provides a RESTfull HTTP DELETE enpoint /files/{file_id}.
+- Upon receiving and validating the deletion request, it publishes an event
```

Comment:
What does validation mean in this context, checking the access token?
This should be rephrased to be a bit more specific